app/assets/stylesheets/notes.scss

```diff
@@ -0,0 +1,19 @@
+.name-date-time {
```

Comment:
It looks like we're basically recreating the existing table styling on a div class rather than a table. What do you think about kind of splitting the difference here and just using existing th and td elements in the `notes_table`? Something like:

```
...
      <% unless note.new_record? %>
        <tr>
          <th><%= note.created_by.name %> <%= note.created_at.display_date %> <%= note.created_at.display_time %></th>
        </tr>
        <tr>
          <td><%= note.full_text %></td>
        </tr>
      <% end %>
...
```

As you can probably guess by the scss patterns, we aren't in that part of the stack too often! So I'm inclined to keep as many changes in the html.erb templates as possible. Your thoughts?